R/logging.R

```diff
@@ -31,6 +31,7 @@
 #' )
 #' @export
 message_wrap <-
+  # TODO melodie; not in logging_melodie
```

Comment:
@EmilHvitfeldt These are my thoughts, but you are way more familiar with these so I defer to you on the next PR